rice-middleware/impl/src/main/java/org/kuali/rice/kew/impl/document/search/DocumentSearchCriteriaBoLookupableHelperService.java

```diff
@@ -942,13 +942,17 @@ protected void executeColumnCustomization(ResultRow resultRow, DocumentSearchRes
         populateCustomColumns(customColumns, searchResult);
 
         // if there is an action custom column, always put that before any other field
+        List<Column> columnsToRemove = new ArrayList<Column>();
```

Comment:
This works fine, but a List isn't really necessary.  There is only one action column and the loop below exits on the first action.

Could be changed to:
```
Column columnToRemove = null
...
if (columnToRemove != null) {
    customColumns.remove(columnToRemove);
}
```